From 835e48074834fb5b888d53b00f84967701d6d78a Mon Sep 17 00:00:00 2001
From: Michael Pyne <mpyne@kde.org>
Date: Sun, 23 Jul 2017 11:37:10 -0400
Subject: [PATCH] Require sys/types.h for dev_t unconditionally.

Fixes a build error with musl libc.  Turns out this is the required
header for dev_t, and not just for FreeBSD.  That it works without on
glibc is just an accident.

Differential Revision: https://phabricator.kde.org/D6596

(cherry picked from commit 2382c3f8d3669c473130f4baefb68d244dcb5cbc)
---
 3rdparty/solid-lite/backends/udisks2/udisksblock.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/3rdparty/solid-lite/backends/udisks2/udisksblock.h b/3rdparty/solid-lite/backends/udisks2/udisksblock.h
index 87c358099a..7b7305d73e 100644
--- a/3rdparty/solid-lite/backends/udisks2/udisksblock.h
+++ b/3rdparty/solid-lite/backends/udisks2/udisksblock.h
@@ -24,6 +24,8 @@
 #include <solid-lite/ifaces/block.h>
 #include "udisksdeviceinterface.h"
 
+#include <sys/types.h> // dev_t
+
 namespace Solid
 {
 namespace Backends
